translations/pt_BR/README.md

```diff
@@ -0,0 +1,336 @@
+- IPNS, um sistema de nomes inspirado em SFS
+- namespace global baseado em PKI
+- serve para construir cadeias de confiança
+- compatível com outros NSes
+- pode mapear DNS, .onion, .bit, etc para IPNS
+
+## Como o IPFS Funciona?
+
+Para saber mais sobre como funciona o IPFS, dê uma olhada nos [Papers](#ipfs-papers) ou [Conversas](#ipfs-talks). Você também pode explorar as [Especificações](https://github.com/ipfs/specs) por escrito.
+
+### Papers IPFS
+
+- [IPFS - Content Addressed, Versioned, P2P File System (draft 3)](https://github.com/ipfs/ipfs/blob/master/papers/ipfs-cap2pfs/ipfs-p2p-file-system.pdf?raw=true)
+- [Especificações](https://github.com/ipfs/specs) (em progresso)
+- veja também: https://github.com/ipfs/papers
+
+### Conversas IPFS
+
+Esta é uma breve seleção de palestras introdutórias. Com o tempo, coletaremos mais.
+
+- 2014-07-21 - [IPFS: The Permanent Web](https://www.youtube.com/watch?v=Fa4pckodM9g) at [Sourcegraph](https://sourcegraph.com/) (primeira conversa pública)
+- 2015-02-20 - [IPFS Alpha Demo](https://www.youtube.com/watch?v=8CMxDNuuAiQ)
+- 2015-06-03 - [IPFS Hands on Introduction](https://www.youtube.com/watch?v=h73bd9b5pPA) at Ethereum SV Meetup
+- **2015-10-22 - [IPFS: The Distributed, Permanent Web](https://www.youtube.com/watch?v=HUVmypx9HGI) no Seminário de Stanford (melhor visão geral do projeto)**
+- 2016-09-14 - [Distributed Apps with IPFS](https://www.youtube.com/watch?v=jONZtXMu03w)
+- 2016-10-22 - [The Decentralized Web, IPFS and Filecoin](https://www.youtube.com/watch?v=cU-n_m-snxQ)
+
+Veja a lista completa em [IPFS Media](https://ipfs.io/media/)
+
+## Mais Sobre IPFS
+
+O projeto IPFS busca evoluir a infraestrutura da Internet e da Web, com muitas coisas que aprendemos com sistemas de sucesso, como o [Git](https://git-scm.com/), [BitTorrent](http://bittorrent.org/), [Kademlia](https://en.wikipedia.org/wiki/Kademlia), [Bitcoin](https://bitcoin.org/), e muitos, muitos outros. Esse é o tipo de coisa que teria saído no ARPA/DARPA/IETF/BellLabs em outra época. IPFS é um projeto Open Source Livre, com centenas de colaboradores.
+
+### Estado Atual do IPFS
+
+**O IPFS é um trabalho em andamento!**
+
+Observe que o IPFS é um trabalho em andamento. É um plano ambicioso para tornar a Internet mais livre, aberta, segura e de alto desempenho. Baseia-se nas boas ideias de numerosos sistemas distribuídos testados em campo.
+
+Hoje, há uma implementação principal do protocolo IPFS (em [Go](https://github.com/ipfs/go-ipfs)) com [mais projetos a caminho](#protocol-implementations) ([JavaScript](https://github.com/ipfs/js-ipfs) e [Python](https://github.com/ipfs/py-ipfs)).
+
+### Distribuição Alpha
+
+In February of 2015, the [go-ipfs](https://github.com/ipfs/go-ipfs) implementation was released as an "Alpha Distribution". Since then, go-ipfs has been making regular releases on the road towards Beta. Both [js-ipfs](https://github.com/ipfs/js-ipfs) and [py-ipfs](https://github.com/ipfs/py-ipfs) are in progress.
```

Comment:
This seems to be duplicated - once in english and then the version in Portuguese